litestar/controller.py

```diff
@@ -143,7 +143,7 @@ class Controller:
     """A sequence of string tags that will be appended to the schema of all route handlers under the controller."""
     security: Sequence[SecurityRequirement] | None
     """A sequence of dictionaries that to the schema of all route handlers under the controller."""
-    signature_namespace: dict[str, Any]
+    signature_namespace: Mapping[str, Any]
```

Comment:
Breaking as well if users rely on this being a dict

Reply:
Reverted